### ISSUE TITLE
Theme makes Ubuntu MATE Welcome and Software Boutique unusable

### ISSUE THREAD

**Issue by segadude** [luigifab/human-theme#17]
Applying this theme makes the Ubuntu MATE Welcome and Software Boutique applications unusable. It causes the background color to change to black so you can't see any of the options. This renders these applications unusable as a result. I have attached screenshots of what these applications look like with the default Ubuntu MATE theme and what it looks like with this human theme. This is on Ubuntu MATE 20.04.4 LTS with MATE 1.24.0.

![Screenshot at 2022-08-18 10-31-03](https://user-images.githubusercontent.com/1498561/185422062-40fc2be2-946c-46bd-bfa5-d5fe92bfc5f2.png)
Software Boutique with the default theme

![Screenshot at 2022-08-18 10-31-57](https://user-images.githubusercontent.com/1498561/185422051-3e572700-5cb8-4b7d-87ae-0fce641490d1.png)
Software Boutique with the human theme

![Screenshot at 2022-08-18 10-32-44](https://user-images.githubusercontent.com/1498561/185422060-f1546959-48c6-4e58-bde6-df80ea23c66e.png)
Ubuntu MATE Welcome with the default theme

![Screenshot at 2022-08-18 10-33-32](https://user-images.githubusercontent.com/1498561/185422061-51b1ac02-4eaa-4ec5-af49-e92f5133adb6.png)
Ubuntu MATE Welcome with the human theme

**Comment by luigifab** [comment]
Found! This is because I don't use keywords used by them. [Here](https://github.com/ubuntu-mate/ubuntu-mate-welcome-legacy/blob/master/ubuntu-mate-welcome):
```python
bg_color = _get_color(style_context, "base_color", "theme_bg_color")
text_color = _get_color(style_context, "fg_color", "theme_fg_color")
section_bg_color = _get_color(style_context, "dark_bg_color", "theme_bg_color")
section_text_color = _get_color(style_context, "dark_fg_color", "theme_fg_color")
selected_bg_color = _get_color(style_context, "selected_bg_color", "theme_selected_bg_color")
selected_text_color = _get_color(style_context, "selected_fg_color", "theme_selected_fg_color")
button_bg_color = _get_color(style_context, "button_bg_color", "theme_bg_color")
button_text_color = _get_color(style_context, "text_color", "theme_fg_color")
```

Add the following into for example gtk3 mate.css file of the theme (ok, not tested with latest theme release, perhaps this will not work):
```css
@define-color theme_bg_color @theme_bg;
@define-color theme_fg_color @theme_fg;
@define-color theme_selected_bg_color @theme_selected_bg;
@define-color theme_selected_fg_color @theme_selected_fg;
```

Perhaps not perfect, but it's a start!

![image](https://user-images.githubusercontent.com/31816829/189498824-e20cbdab-1f0b-4c01-8d3c-2f88f2b9a764.png)

**Comment by luigifab** [comment]
Fixed in v2.0.0.
